src/js/view/controls/components/timeslider.js

```diff
@@ -269,6 +269,7 @@ class TimeSlider extends Slider {
     }
 
     addCues(model, cues) {
+        this.resetChapters();
```

Comment:
Is the `else` needed, now that `this.resetChapters();` happens at the start of the function?

         } else {
             this.resetChapters();

Reply:
No. @waxidiotic please remove the `else`.

Reply:
Done. Thanks @Walfisch - Must have missed that.